ads_plugin.py

```diff
@@ -364,11 +366,11 @@ def on_load_mask_button(self, event):
         myelin_mask = params.intensity['binary'] * np.array(myelin_mask, dtype=np.uint8)
 
         # Load the masks into FSLeyes
-        axon_outfile = self.ads_temp_dir / (image_name + "-axon.png")
+        axon_outfile = self.ads_temp_dir / (image_name + "-axon-manual_pred.png")
```

Comment:
```suggestion
        axon_outfile = self.ads_temp_dir / (image_name + "-axon.png")
```

If I understand correctly, this suffix is used in the "overlay" section of the GUI. i.e., when we load an axonmyelin mask, it names the layer with the `filename-axon` and `filename-myelin`. Adding "manual_pred" is redundant, see:
![image](https://user-images.githubusercontent.com/54086142/146605446-d46f4c8f-2476-41ae-86cd-cae2bd854aad.png)

So those PNG may only be "internal" in fsleyes and never seen by the user.

Reply:
I'll let @Stoyan-I-A comment here, because I know he's setup some logic in the plugin (i.e. automatically call certain types of masks or fetch them for certain purposes) and I don't want to mess things up by changing these lines.

Reply:
Every file that is saved under `ads_temp_dir` are deleted at the end of the session.
I will look into the rest of the logic with the suffixes to try to make it more consistant with the naming convention we have in ADS.